File: Bootcamp_Coding_Exercises.R
# author: Monish Dadlani (mkd7q@virginia.edu)
# date : 12-26-2019

rm(list=ls())

# Exercise 1
current_time <- Sys.time()
print(current_time)

# Exercise 2
start <- Sys.time()
data(cars)
attach(cars)
cars$time <- cars$dist/cars$speed
end <- Sys.time()
elapsed_time <- end-start
print(elapsed_time)

# Exercise 3
word <- readline(prompt = "Enter a word")
print(word)

# Exercise 4
word <- readline(prompt = "Enter a word")
letters_only <- function(x) {!grepl("[^A-Za-z]", x)}
if (letters_only(word)) {
  print(word)
} else {
  readline(prompt = "Enter another word")
}

#Exercise 5
first_name <- readline(prompt = "Enter your first name")
last_name <- readline(prompt = "Enter your last name")
college <- readline(prompt = "Enter the name of your college")
degree <- readline(prompt = "Enter degree pursuing")
residence <- readline(prompt = "Enter your city of residence")    
student_info = list(first_name,last_name,college,degree,residence)
for(i in student_info){print(i)}

File: Bootcamp_Coding_Exercises.py
# -*- coding: utf-8 -*-
"""
Created on Mon Dec 23 14:35:25 2019

@author: dadlamk
"""

# Exercise 1
from datetime import datetime
current_time = datetime.now()
current_time = current_time.strftime("%m/%d/%y %H:%M:%S")
print(current_time)
print(datetime.now())

# Exercise 2
from datetime import datetime
start_time = datetime.now()
x=3
y=5
x+y
x-y
x*y
x//y
end_time = datetime.now()
elapsed_time = end_time - start_time
print(elapsed_time)

# Exercise 3
word = input("Enter a word")
print(word)

# Exercise 4
word = input("Enter a word")
if word.isalpha():
    print(word)
else:
    input("Enter another word")
    
# Exercise 5
first_name = input("Enter your first name")
last_name = input("Enter your last name")
college = input("Enter the name of your college")
degree = input("Enter degree pursuing")
residence = input("Enter your city of residence")    
student_info = [first_name,last_name,college,degree,residence]
for i in student_info:
    print(i)